packages/client/hmi-client/src/workflow/ops/model-from-code/tera-model-from-code.vue

```diff
@@ -276,12 +277,29 @@ async function handleCode() {
 	isProcessing.value = true;
 
 	if (clonedState.value.modelFramework === ModelFramework.Petrinet) {
-		const modelId = await codeToAMR(
-			props.node.inputs[0].value?.[0],
-			'temp model',
-			'temp model description',
-			true
-		);
+		const codeContent = allCodeBlocks.value
+			.filter((block) => block.includeInProcess)
+			.reduce((acc, block) => `${acc}${block.asset.codeContent}\n`, '');
+		const file = new File([codeContent], 'tempFile');
```

Comment:
put the code blocks all in 1 file and upload it to a "temporary" (public) code-asset which will be used to call the KM code-to-amr endpoint